DER/DerivativesContracts/Swaps.rdf

```diff
@@ -212,7 +242,7 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-der-drc-swp;ReturnLeg">
-		<rdfs:subClassOf rdf:resource="&fibo-der-drc-swp;SwapLeg"/>
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-swp;FloatingLeg"/>
```

Comment:
if the ReferenceIndex is fixed surely this will be fixed

Reply:
The return leg is considered the floating leg of a return swap.  The strike leg is the fixed leg.  The return may or may not be based on a fixed rate of return, but often is not.